federation-js/src/service/printSubgraphSchema.ts

```diff
@@ -54,7 +58,7 @@ type Options = {
  *
  *    - commentDescriptions:
  *        Provide true to use preceding comments as the description.
- *
+ * @deprecated Use `printSubgraphSchema` instead.
  */
 export function printSchema(schema: GraphQLSchema, options?: Options): string {
```

Comment:
Maybe turn this around: make `printSubgraphSchema` the main function, and alias it as `printSchema` for backwards compatibility?

Reply:
Ah, yes of course 😄

Reply:
5227ad5